TF_Settings_Web/src/TouchFree/Configuration/ConfigurationManager.ts

```diff
@@ -62,14 +62,14 @@ export class ConfigurationManager {
     public static RequestConfigFileChange(
         _interaction: Partial<InteractionConfig> | null,
```

Comment:
All the changes to the TouchFree Tooling applied from here down in the listed changes need to be replicated across to the "main" version of the TouchFree tooling before we end up with really divergent versions.

Reply:
Copying one atop the other they seem to have quite a few differences: 
![image](https://user-images.githubusercontent.com/56393090/183406677-fad5b983-fdb5-4e20-86f3-b1ef1d1bd113.png)